day3-b.js

```diff
@@ -0,0 +1,33 @@
+//**Hello Ramin this is part 1 of day 3 :D **//
+//i am running this online to get the input here : https://www.codechef.com/ide //
+process.stdin.resume();
+process.stdin.setEncoding('utf8');
+
+
+process.stdin.on('data', function(data) { 
+    
+    var test = data.split("\n")
```

Comment:
In modern JavaScript we are using `let` and `const` instead of `var`. We only use `var` if really needed.

```suggestion
    const test = data.split("\n")
```